From 0428fc5c8b099944efe267f29cfe6b7e0a5b7554 Mon Sep 17 00:00:00 2001
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Tue, 6 Apr 2010 17:37:33 -0300
Subject: [PATCH] Set CONFIG_S3_RESUME_VGA_INIT to 1

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <20100406173733.GB19640@blackpad.lan.raisama.net>
Patchwork-id: 8373
O-Subject: [RHEL6 PATCH seabios] Set CONFIG_S3_RESUME_VGA_INIT to 1
Bugzilla: 567910
RH-Acked-by: Zachary Amsden <zamsden@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Gleb Natapov <gleb@redhat.com>

Bugzilla: 567910

>From https://bugzilla.redhat.com/show_bug.cgi?id=567910:

> Eduard Benes 2010-02-24 05:03:08 EST
> According to conversation in bug 545808, seabios configuration parameter
> CONFIG_S3_RESUME_VGA_INIT should default to 1 for RHEL 6. This way we can make
> KVM bios to run vga rom on S3 resume without running the vbetool (which won't
> be present) inside a virt guest.

Please ACK.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 src/config.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 src/config.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/config.h b/src/config.h
index 6297a48..afa0612 100644
--- a/src/config.h
+++ b/src/config.h
@@ -110,7 +110,7 @@
 // Support S3 resume handler.
 #define CONFIG_S3_RESUME 1
 // Run the vga rom during S3 resume.
-#define CONFIG_S3_RESUME_VGA_INIT 0
+#define CONFIG_S3_RESUME_VGA_INIT 1
 // define it if the (emulated) hardware supports SMM mode
 #define CONFIG_USE_SMM 1
 // Maximum number of map entries in the e820 map
-- 
1.7.0.3

